internal/kafka/internal/presenters/kafka.go

```diff
@@ -29,6 +29,7 @@ func ConvertKafkaRequest(kafkaRequestPayload public.KafkaRequestPayload, dbKafka
 
 	kafka.BillingCloudAccountId = shared.SafeString(kafkaRequestPayload.BillingCloudAccountId)
 	kafka.Marketplace = shared.SafeString(kafkaRequestPayload.Marketplace)
+	kafka.BillingModel = shared.SafeString(kafkaRequestPayload.BillingModel)
```

Comment:
Based on https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0844d0bbbc615c5a17105c5a2878236bee686838/openapi/kas-fleet-manager.yaml#L1084 the field can be `null` and with this logic here, the `BillingModel` will be empty. That's desired because we want to keep the old behavior. But I do not see a code change on the old behavior that assigns the `BillingModel` depending on whichever that was picked. The advantage of doing this is that, we'll be sure that this field is always set in the database. This is something that we might be missing at the current PR state.

Reply:
Yes, good point. I'll add that assignment.

Reply:
Thanks @pb82

Reply:
That's done now